nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,106 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    Directory,
+    traits,
+    OutputMultiPath
+)
+import os
+
+class ICA_AROMAInputSpec(CommandLineInputSpec):
+    feat_dir = Directory(exists=True, mandatory=True,
+                        argstr='-feat %s', 
+                        xor=['in_file', 'mat_file', 'fnirt_warp_file', 'motion_parameters'],
+                        desc='If a feat directory exists and temporal filtering '
+                        'has not been run yet, ICA_AROMA can use the files in '
+                        'this directory.')
+    in_file = File(exists=True, mandatory=True, 
+                  argstr='-i %s', xor=['feat_dir'], 
+                  desc='volume to be denoised')
+    out_dir = Directory('out', mandatory=True,
+                       argstr='-o %s',  
+                       desc='output directory')
+    mask = File(exists=True, argstr='-m %s', xor=['feat_dir'], 
+               desc='path/name volume mask')
+    dim = traits.Int(argstr='-dim %d',
+                    desc='Dimensionality reduction when running '
+                    'MELODIC (defualt is automatic estimation)')
+    TR = traits.Float(argstr='-tr %.3f',
+                     desc='TR in seconds. If this is not specified '
+                     'the TR will be extracted from the '
+                     'header of the fMRI nifti file.')
+    melodic_dir = Directory(exists=True, argstr='-meldir %s',
+                           desc='path to MELODIC directory if MELODIC has already been run')
+    mat_file = File(exists=True, mandatory=True,
+                   argstr='-affmat %s', xor=['feat_dir'],
+                   desc='path/name of the mat-file describing the '
+                   'affine registration (e.g. FSL FLIRT) of the '
+                   'functional data to structural space (.mat file)')
+    fnirt_warp_file = File(exists=True, mandatory=True,
```

Comment:
This may be true. I honestly haven't dug into their documentation to be sure what all is actually mandatory.

Here is their argparse setup: https://github.com/rhr-pruim/ICA-AROMA/blob/master/ICA_AROMA.py#L27-L37